packages/embed/package.json

```diff
@@ -20,7 +20,9 @@
     "test": "jest",
     "cy:run": "yarn cypress run --browser chrome --headless",
     "cy:open": "yarn cypress open",
-    "test:functional": "start-server-and-test demo 6006 cy:run"
+    "cy:visual": "yarn cypress run --spec e2e/spec/visual/**/* --browser chrome",
```

Comment:
- Lets rename `cy:run` to `cy:functional` and add `--spec` param as well to make sure it runs only functional tests.
- You also need to move functional spec to its own dir.
- We need to run visual tests in Github Actions on PR.
- Please write at least one test for demonstration purposes. You can open stories directly without the storybook UI like we do for functional tests now:
```
cy.visit('/iframe.html?id=embed-popup--popup&viewMode=story')
```